tests/discover/data/distgit_test_plugin.py

```diff
@@ -17,14 +16,14 @@ class TestDistGit(DistGitHandler):
     usage_name = "TESTING"
     server = f"http://localhost:{SERVER_PORT}"
 
-    def url_and_name(self, cwd='.'):
-        with open(os.path.join(cwd, MOCK_SOURCES_FILENAME)) as f:
-            data = f.read()
+    def url_and_name(self, cwd=None):
```

Comment:
Was there ever a consensus about the use of `https://docs.python.org/3/library/os.html#os.PathLike` or something to annotate 'pathlike'?

Reply:
Nope, never discussed there, we use `tmt.utils.Path` everywhere.